source/code/source/playfab/PlayFabEventBuffer.cpp

```diff
@@ -35,7 +35,7 @@ namespace PlayFab
 
         // First event is just a stub which is used only to maintain the initial consistency of m_head/m_tail.
         // It is immediately considered already consumed.
-        auto index = eventIndex->load(std::memory_order_relaxed);
+        unsigned long long index = eventIndex->load(std::memory_order_relaxed);
```

Comment:
>unsigned long long  [](start = 8, length = 19)

eventIndex is a std::shared_ptr<std::atomic<uint64_t>>

comments aside on how necessary that is, that tells me that the type returned by `load` is u64 not `unsigned long long` (they may evaluate to equivalent types on most compilers but it's unnecessarily confusing to change the type here)